### ISSUE TITLE
Help - how to use command `set_destination` in navit.xml

### ISSUE THREAD

**Issue by xuniuer** [navit-gps/navit#1058]
Hi guys!

This is not an issue but a question.

In my application, when `clicking` screen, I need to use the position as destination. Of course the beginning position is from local GPS. So I think if I can set the position as destination will be OK. But how to use command `set_destination` in navit.xml file. I check the link: https://wiki.navit-project.org/index.php/OSD, no tips.  

Thanks!

**Comment by edd-cgfan** [comment]
Hi,

In the main menu I have an icon that sets the destination with an onclick command:
`<img src='school' onclick='navit.set_destination("mg:0x1241f0 0x672bf2","WR Parkplatz");back_to_map();redraw_map()'><text>WR_HS</text></img>`

`mg:0x1241f0 0x672bf2` was taken from bookmark file.

Maybe this helps.

**Comment by xuniuer** [comment]
@edd-cgfan Yes, it helps. If I have known destination positions, in fact, I have in my application, the solution is fine. 

Thanks!

**Comment by phice1** [comment]
I would like to do the same thing, however I would like to not have to leave the screen to enter a menu, I have the waypoint_flag set so that 'plan with waypoints' is enabled, so would simply like the on_click to take the point on the map and have the 'set_destination' create a waypoint at the coordinates of the click.
